lib/srv/desktop/rdp/rdpclient/src/lib.rs

```diff
@@ -335,6 +368,7 @@ fn connect_rdp_inner(
         rdp_client: Arc::new(Mutex::new(rdp_client)),
         tcp_fd,
         go_ref,
+        tcp: shared_tcp2,
```

Comment:
nit to remove gross variable name `shared_tcp2`

```suggestion
        tcp: shared_tcp.clone(),
```

Reply:
I'd really like to get rid of this variable but I can't since `shared_tcp` is already moved into the `mcs` client by this point and we have no access to it.

Reply:
Oh ok gotcha